stripe/functions/index.js

```diff
@@ -63,6 +63,22 @@ exports.createStripeCustomer = functions.auth.user().onCreate(event => {
   });
 });
 
+// Add a payment source (card) for a user by writing a stripe payment source token to Realtime database
+exports.addPaymentSource = functions.database.ref('/stripe_customers/{userId}/cards/{token}').onWrite(event => {
```

Comment:
@jamesdaniels yea you're right, I missed one there. Should be `/stripe_customers/{userId}/cards/token`. I'll make this change.

I was going off of what was returned in [customer creation](https://github.com/firebase/functions-samples/blob/e9490d7e8df49a53eb9da44e5865044dc7f90aba/stripe/functions/index.js#L62). Seems to me you are setting the Stripe customer id at this location?